lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -39,6 +40,10 @@ def fragment_cache(cached_hash, non_cached_hash)
           JsonApi::FragmentCache.new().fragment_cache(root, cached_hash, non_cached_hash)
         end
 
+        def default_key_format
+          :dasherize
```

Comment:
Awesome! Have you benchmarked the performance impact of mutating the hash by default?